integration/message_test.go

```diff
@@ -0,0 +1,71 @@
+package integration_test
+
+import (
+	"log"
+	"sync"
+	"testing"
+
+	"github.com/Shopify/sarama"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+
+	"github.com/digicatapult/wasp-simple-h264-payload-parser/kafka"
+	"github.com/digicatapult/wasp-simple-h264-payload-parser/wasp"
+)
+
+func TestWaspTransformRelay(t *testing.T) {
+	inTopicName := "videos"
+	outTopicName := "videos-transformed"
+	kafkaBrokers := []string{"localhost:9092"}
```

Comment:
Are you sure?

Surely intopic name is `payloads.simpleH264` and outTopic is `video` ?